sdk/resourcemanager/Azure.ResourceManager/tests/Scenario/SubscriptionOperationsTests.cs

```diff
@@ -182,6 +182,7 @@ await foreach (var feature in subscription.GetFeaturesAsync())
         }
 
         [RecordedTest]
+        [Ignore("Re-record of this test will cause github not able to show file diffs and run CI")]
         public async Task ValidateResourceInRestApi()
```

Comment:
The test passed locally but checking in 2 re-recorded files with more tahn 400 thousands diffs each caused CI unable to be triggered. It might be a github glitch, but I prefer to add back this test in another PR. We probably need to remove the recordings in one PR, then reenable the test and add back the recordings in another PR if re-recording still cause CI not to run.